File: src/Demo.DbRefreshManager.Common/Config/Concrete/FrontendConfig.cs
namespace Demo.DbRefreshManager.Common.Config.Concrete;

/// <summary>
/// Конфигурация frontend.
/// </summary>
public class FrontendConfig
{
    /// <summary>
    /// Url метода просмотра списка обновлённых объектов.
    /// </summary>
    public string ObjectsListUrl { get; set; } = string.Empty;

    /// <summary>
    /// Url интрукции к менеджеру.
    /// </summary>
    public string InstructionUrl { get; set; } = string.Empty;
}

File: src/Demo.DbRefreshManager.WebUI/src/shared/types/api/frontendConfig/frontendConfig.ts
/** Конфигурация frontend. */
export type FrontendConfig = {
    /** Url метода просмотра списка обновлённых объектов. */
    objectsListUrl: string
    /** Url интрукции к менеджеру. */
    instructionUrl: string
};

/** Массив ключей типа. */
export const FrontendConfigKeys: Readonly<Array<keyof FrontendConfig>> = [
    'objectsListUrl',
    'instructionUrl'
];

export default FrontendConfig;